src/app/components/pages/SetAssignments.tsx

```diff
@@ -606,7 +605,7 @@ export const SetAssignments = () => {
                                             {boards.boards
                                                 .filter(board => board.title && board.title.toLowerCase().includes(boardTitleFilter.toLowerCase())
                                                     && (formatBoardOwner(user, board) == boardCreator || boardCreator == "All")
-                                                    && (boardSubject == "All" || (determineGameboardSubjects(board).includes(boardSubject.toLowerCase()))))
+                                                    && (boardSubject == "All"))
```

Comment:
I think this condition can probably be removed. see the comments below

Reply:
Removed boardSubject and associated "generateGameboardSubjectHexagons"